Ntare22/MyDiary-Andela
# #169518867 create an account db

> #### What does this PR do?
* connects to the database and user is able to sign up, and saved in the database.
> #### Description of Task to be completed
* set up postgres db
* adjust signup method
* adjusted usertests file
* adjust travis.yml
> #### How should this be manually tested?
* clone this repo
* create .env file
* set your pwdSalt, secrectKey and dbURL (postgres credentials link) in the .env file
* enter "npm run dev" in your commnad line.
* open postman, add "localhost:3000/api/v2/auth/signup" in the address bar
* choose POST
* create JSON file with firstName, lastName, email, and password credentials
* click send
> #### Any background context you want to provide?
* N/A
> #### What are the relevant pivotal tracker stories?
[#169518867](https://www.pivotaltracker.com/story/show/169522775)
> #### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30793016/68087311-4421f300-fe5d-11e9-9590-44ac6ab1dc71.png)
![image](https://user-images.githubusercontent.com/30793016/68087314-53a13c00-fe5d-11e9-9887-ad32979379e4.png)
